README.md

```diff
@@ -40,82 +67,141 @@ Install application dependencies:
 npm install
 ```
 
-### Development
+---
 
-To run the application in `development` mode run:
+### Test
+
+Test continously:
 
 ```bash
-npm run dev
+npm run test:watch
 ```
 
-### Local JSON API
+---
 
-Whilst the APIs are being developed this app uses a local JSON mock API. To start this locally run:
+## Running
+
+### Directly
+
+To run the application in `development` mode run:
 
 ```bash
-npm run mockApi
+npm run dev
 ```
 
-### Production
+### Docker Compose
 
-To mimic the application running in `production` mode locally run:
+A local environment is provided with:
+
+- Redis
+- CDP DEFRA ID stub
 
 ```bash
-npm start
+docker compose up --build -d
 ```
 
-### Npm scripts
+## Integrate
 
-All available Npm scripts can be seen in [package.json](./package.json)
-To view them in your command line run:
+More details to come
 
-```bash
-npm run
-```
+To integrate locally, see [running](#running) above).stub.infra-dev.cdp-int.defra.cloud/cdp-defra-id-stub/.well-known/openid-configuration'
 
-## Docker
+### OIDC URL
 
-### Development image
+To integrate in an environment, configure your app's **OIDC Configuration URL** to be one of:
 
-Build:
+- Local: `http://localhost:3200/cdp-defra-id-stub/.well-known/openid-configuration`
+- Dev: `https://cdp-defra-id-stub.dev.cdp-int.defra.cloud/cdp-defra-id-stub/.well-known/openid-configuration`
+- Test: `https://cdp-defra-id-stub.test.cdp-int.defra.cloud/cdp-defra-id-stub/.well-known/openid-configuration`
+- Perf test: `https://cdp-defra-id-stub.perf-test.cdp-int.defra.cloud/cdp-defra-id-stub/.well-known/openid-configuration`
 
-```bash
-docker build --target development --no-cache --tag cdp-defra-id-stub:development .
-```
+### Client secret
 
-Run:
+(This will change soon)
 
-```bash
-docker run -p 3000:3000 cdp-defra-id-stub:development
+Set the **DEFRA ID Client Secret** to `test_value`6
```

Comment:
Is the 6 part of the secret? im guessing it might be a typo!

Reply:
Whoops